src/SearchResult.test.tsx

```diff
@@ -97,20 +71,15 @@ describe("SearchResult", () => {
   describe("without URL", () => {
     const searchResultNoUrl = {
       ...MOCK_SEARCH_RESULT,
-      url: undefined,
+      url: undefined
     };
     it("should render a div if no URL was provided", () => {
-      render(
-        <SearchResult searchResult={searchResultNoUrl} isSelected={false} />
-      );
-      const divWithTextContent = screen.getByText(
-        MOCK_SEARCH_RESULT.snippet.text
-      );
-      const anchor = divWithTextContent.closest("a");
-      const wrappingDiv = divWithTextContent.closest("div");
+      render(<SearchResult searchResult={searchResultNoUrl} isSelected={false} />);
+      const anchor = screen.queryByTestId("vectara-react-search-result-link");
+      const div = screen.getByTestId("vectara-react-search-result-div");
```

Comment:
I think a more semantic term in the testId would be "wrapper" instead of "div".

Reply:
https://github.com/vectara/react-search/pull/22/commits/8711b05a1fd7913806accfe756898908baeaf09f